domain/src/main/java/org/oppia/android/domain/question/QuestionAssessmentProgressController.kt

```diff
@@ -619,7 +619,8 @@ class QuestionAssessmentProgressController @Inject constructor(
   private fun ControllerState.computeBaseCurrentEphemeralState(): EphemeralState =
     progress.stateDeck.getCurrentEphemeralState(
       hintHandler.getCurrentHelpIndex().value,
-      oppiaClock.getCurrentTimeMs()
+      oppiaClock.getCurrentTimeMs(),
+      true
```

Comment:
This needs to include the parameter name to provide context as to what the ``true`` value corresponds to, e.g. ``parameterName = true``

Reply:
done